.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what is broken -->
+<!--- If suggesting a change/improvement tell us what is missing from the SDK as it is currently -->
+
+## Expected Behavior
+<!--- If describing a bug, tell us how the SDK should behave without the bug -->
+<!--- If you're suggesting changes, outline your idea here -->
+
+## Possible Solution
+<!--- Optional, but helpful! Use this field to suggest bug solutions or change impementation details. Feel free to get technical. -->
```

Comment:
Can you move the

> Optional, but helpful!

part to the end? I _really_ want folks to do this part 😄

Reply:
+1